tests/sample/data_usage/associatetoflow.c

```diff
@@ -0,0 +1,27 @@
+#include <stdbool.h>
+#include <stdint.h>
+#include "ebpf_helpers.h"
+#include "ebpf_nethooks.h"
+
+#pragma clang section data = "maps"
+ebpf_map_definition_t app_map = {
+    .size = sizeof(ebpf_map_definition_t),
+    .type = EBPF_MAP_TYPE_HASH,
+    .key_size = sizeof(five_tuple_t),
+    .value_size = sizeof(uint64_t),
+    .max_entries = 500};
+
+
+#pragma clang section text = "flow"
+int
+AssociateFlowToContext(flow_md_t* ctx)
+{
+    if (ctx->flow_established_flag) {
+        ebpf_map_update_element(&app_map, &ctx->five_tuple, &ctx->app_id, 0);
```

Comment:
Does the existing framework support testing map operations without having the program info provider?